.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Maven publish
```

Comment:
I'd put limitations on this, for example only on push/tag and on the `master` branch